File: research/check_time.c
#include "stdio.h"
#include "stdlib.h"
#include "time.h"

int check1(int n)
{
	clock_t st;
	st = clock();
	int summ = 22;
	for (int i = 0; i <= n; i++)
	{
		summ *= 22;
		summ /= 22;
	}
		
	return clock() - st;	
}

int main()
{
	printf("%d\n", check1(50000000));
	return 0;
}

File: research/check_time.py
from time import time, process_time, clock

def check1():
    st = time()
    summ = 0
    for i in range(50000000):
        summ += 1
    return time() - st

def check2():
    st = process_time()
    summ = 0
    for i in range(50000000):
        summ += 1
    return process_time() - st
